packages/amplify-cli-shared-interfaces/API.md

```diff
@@ -1,86 +1,88 @@
-## API Report File for "amplify-cli-shared-interfaces"
+## API Report File for "@aws-amplify/amplify-cli-shared-interfaces"
 
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+
 // @public (undocumented)
 export interface ICommandInput {
-  // (undocumented)
```

Comment:
should this formatting be reverted?

Reply:
I didn't change this formatting, just reran the verify api-extract script